### ISSUE TITLE
Call out specific action for resource star finding

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/parliament#117]
The finding `Unnecessary use of Resource *` will mention something like `actions: ['ec2:CreateVolume', 'ec2:DescribeInstances']`.  In this example, `ec2:CreateVolume` can be scoped down further, but `ec2:DescribeInstances` cannot, so I should not mention that action.

**Comment by kmcquade** [comment]
Just an idea - we *could* perhaps remove that specific community auditor entirely, given that I target that specifically in Cloudsplaining. 

If you want to keep it, I suggest removing the Resource `*` portion for Read/List actions, that way it focuses on modify level blast radius. Otherwise, it is too noisy - imho.

**Comment by dgubitosi** [comment]
I can take a look at this.  I can see the value in this but there's some challenges with identification to consider:
1) The actions[] list really helps the user find the offending section of the policy.  It will no longer exactly match what's in the policy if it only reports the offending actions making it a little more difficult to find in the original policy.  While this is less of a problem now that's sid reporting, but sid's are not mandatory.
2) What about wildcard actions like ec2:* or ec2:Create*?  Should the finding report the expanded list of offending actions?  I can see the value in reporting them but this also makes it more difficult to find the offending section of the policy.

**Comment by 0xdabbad00** [comment]
Those are good points @dgubitosi.  I think Parliament is always going to have a problem with identifying the specific location of an issue until #62 is resolved, which is a big task so I think it's just going to have to be a limitation of this library for the foreseeable future.  I'm in favor of Parliament better identifying "what" the problem is at the cost of identifying "where" the problem is, so for example, I'm in favor of identifying specifically which action is problematic rather than listing all of the actions in the statement, or listing `ec2:*`.  

So given a policy like this:
```
{
  "Version": "2012-10-17",
  "Statement": {
    "Effect": "Allow",
    "Action": "ec2:Create*",
    "Resource": "*",
  }
}
```

I would want a finding that identifies `ec2:CreateVolume`, `ec2:CreateVpcEndpoint` and other actions that can be scoped down further, but not `ec2:CreateVpc` which cannot be scoped further.

**Comment by dgubitosi** [comment]
For discussion: https://github.com/duo-labs/parliament/pull/132

**Comment by 0xdabbad00** [comment]
Closed by #132
